templates/vue/src/components/Lightbox/media/VideoMedia/H5PVideoMedia.vue

```diff
@@ -210,7 +210,11 @@ export default {
          */
         newSettings.caption = {}
       }
-      if (Helpers.isDifferent(newSettings, this.h5pSettings)) {
+      if (
+        this.h5pSettings &&
+        this.newSettings &&
```

Comment:
What was the error here? Why not pass this check into `Helpers.isDifferent` instead? Also, if two passed variables are both undefined or false, they are technically equal.

Reply:
`this.h5pSettings` and` this.newSettings` are sometimes both null, generating an error in the console when Helper.isDifferent hits `const srcKeys = Object.keys(src)`. I could make a statement in Helper.isDifferent like `if(src == null && other == null) {return false}` instead.

Reply:
@odotadot ok I see that the `isDifferent` function currently only checks objects (which is a bad name for that function). We could extend this function to also check more basic variables (and maybe even arrays) as well and check to see if it's an object instead of assuming that it is one. You do not need to check against `null` or anything like that because if you are doing a check with `===` it will automatically check the type and value for you.

Reply:
I think it makes more sense to keep it the way it currently is. If we move the logic into Helpers.isDifferentObject, it will have to return false (to prevent updating the h5psettings), which seems weird (two null objects should be considered the same).